README.md

```diff
@@ -45,10 +47,67 @@ data "healthchecksio_channel" "pagerduty" {
 
 More examples can be found in the [examples directory](./examples).
 
-## Import
+### Import
 
 Checks can be imported using the uuid, e.g.
 
 ```
 $ terraform import healthchecksio_check.my_first_check 760ca858-576a-432b-8b1f-378049d7ce96
 ```
+
+## Development
+
+### Pre-requisites
+- [Go](https://golang.org/) 1.12 or later
+- [Terraform](https://www.terraform.io/) v0.12 or later
+- [Healthchecks.io](https://healthchecks.io/) account and an API key
+- [goreleaser](https://goreleaser.com/) 0.85.1 or later
```

Comment:
Also go-healthchecksio? ;-)

Reply:
True, but we'll handle these kinds of dependencies using go modules. ;)